spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkScan.java

```diff
@@ -103,6 +103,10 @@ protected Types.StructType groupingKeyType() {
 
   protected abstract List<? extends ScanTaskGroup<?>> taskGroups();
 
+  protected SparkReadConf readConf() {
```

Comment:
How is this passed in other cases?

Reply:
https://github.com/apache/iceberg/pull/6651#discussion_r1103847386